aten/src/ATen/native/cuda/int4mm.cu

```diff
@@ -668,15 +821,29 @@ __launch_bounds__(Warps* kWarpSize) void tinygemm_m16n8k16_chunk_kernel(
       // We don't simply accumulate into `c` as this creates a too-strong
       // execution dependency. Instead, we only periodically accumulate into
       // `c`
+#if defined(USE_ROCM)
+      VecT<float, 4> cTmp[2];
+#else
       float4 cTmp[2];
+#endif
```

Comment:
Just curious, shouldn't hipify being able to take care of those? (I assume that's not the only kernel that uses `float4` in the codebase

Reply:
I don't think we can generalize that. 
I had to use `VecT` since `__builtin_amdgcn_mfma_f32_16x16x16bf16_1k()` expected that.